src/routes/safe/components/Balances/Collectibles/index.tsx

```diff
@@ -104,26 +107,31 @@ const Collectibles = (): React.ReactElement => {
 
   if (nftAssetsFromNftTokens.length === 0) {
     return (
-      <Card className={classes.cardOuter}>
-        <div className={classes.cardInner}>
-          <Paragraph className={classes.noData}>{nftLoaded ? 'No NFTs available' : 'Loading NFTs...'}</Paragraph>
-        </div>
-      </Card>
+      <Fragment>
+        {children}
+        <Card className={classes.cardOuter}>
+          <div className={classes.cardInner}>
+            <Paragraph className={classes.noData}>{nftLoaded ? 'No NFTs available' : 'Loading NFTs...'}</Paragraph>
+          </div>
+        </Card>
+      </Fragment>
     )
   }
 
   return (
     <>
       <VirtualizedList
         data={nftAssetsFromNftTokens}
-        itemContent={(_, nftAsset) => {
+        itemContent={(index, nftAsset) => {
           // Larger collectible lists can cause this to be initially undefined
           if (!nftAsset) {
             return null
           }
 
           return (
             <Fragment key={nftAsset.slug}>
+              {index === 0 && children}
```

Comment:
I think it would be easier to understand if we move the children outsite of the VirtualizedList.

e.g.
```
<>
  {children}
  <VirtualizedList ... />
</>

Reply:
VirtualizedList unfortunately needs a fixed height and this banner being closable doesn't make it easy to set it.